src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,38 @@
 package mate.academy
 
+import mate.academy.figure.Circle
+import mate.academy.figure.IsoscelesTrapezoid
+import mate.academy.figure.Rectangle
+import mate.academy.figure.RightTriangle
+import mate.academy.model.Color
+import mate.academy.figure.Square
+
+import mate.academy.service.FigureSupplier
+
+private const val THREE_UNITS = 3.0
+private const val FIVE_UNITS = 5.0
+
 fun main() {
-    // draw all figures here
+    val circle = Circle(Color.WHITE, THREE_UNITS)
+    val isoscelesTrapezoid = IsoscelesTrapezoid(Color.GREEN, THREE_UNITS, FIVE_UNITS, THREE_UNITS)
+    val rectangle = Rectangle(Color.BLUE, THREE_UNITS, FIVE_UNITS)
+    val rightTriangle = RightTriangle(Color.BLACK, THREE_UNITS, FIVE_UNITS)
+    val square = Square(Color.YELLOW, THREE_UNITS)
+
+    println("\nDrawing figure by constructor:")
+    circle.draw()
+    isoscelesTrapezoid.draw()
+    rectangle.draw()
+    rightTriangle.draw()
+    square.draw()
+
+    val figureSupplier = FigureSupplier()
+
+    println("\nDrawing default figure:")
+    figureSupplier.getDefaultFigure().draw()
+
+    println("\nDrawing random figure:")
+    figureSupplier.getRandomFigure().draw()
```

Comment:
Please check task requirements. There should by array or list in the main().
Try to use some relevant loop for handling initialization and draw() methods